internal/input/ledger_range.go

```diff
@@ -0,0 +1,159 @@
+package input
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/go/exp/ingest/ledgerbackend"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+type graphPoint struct {
+	Seq       int64
+	CloseTime time.Time
+}
+
+type graph struct {
+	Backend              *ledgerbackend.HistoryArchiveBackend
+	BeginPoint, EndPoint graphPoint
```

Comment:
Nit: Place struct fields, even if the same type, on different lines for cleaner readability.